specification/eventgrid/data-plane/Microsoft.HealthcareApis/stable/2018-01-01/examples/fhir_resource_created.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "e4c7f556-d72c-e7f7-1069-1e82ac76ab41",
+  "topic": "/subscriptions/4805176c-73e4-4b8a-88cf-ea7d61b176a7/resourceGroups/rg-ga-ui-wsstatus-central/providers/Microsoft.HealthcareApis/workspaces/wsga",
+  "subject": "wsga-fs-ga-wsstatus-r4-01.fhir.azurehealthcareapis.com/Patient/e0a1f743-1a70-451f-830e-e96477163902",
+  "data": {
+    "resourceType": "Patient",
+    "resourceFhirAccount": "wsga-fs-ga-wsstatus-r4-01.fhir.azurehealthcareapis.com",
```

Comment:
same, let's use mocking strings for the FHIR account endpoint, e.g: {your-fhir-account}.fhir.azurehealthcareapis.com